### ISSUE TITLE
[sqlite3] build fails on macOS 11.5.1

### ISSUE THREAD

**Issue by 569b5fc7-560b-4c48-8a4c-79589186cccd** [python/cpython#89160]
BPO | [44997](https://bugs.python.org/issue44997)
--- | :---
Nosy | @ronaldoussoren, @ned-deily, @ambv, @miss-islington, @erlend-aasland, @mcc-devel
PRs | <li>python/cpython#27979</li><li>python/cpython#27994</li><li>python/cpython#27995</li><li>python/cpython#27996</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = <Date 2021-08-27.11:37:42.531>
created_at = <Date 2021-08-25.10:03:29.733>
labels = ['build', '3.8', '3.9', '3.10', '3.11']
title = '[sqlite3] build fails on macOS 11.5.1'
updated_at = <Date 2021-08-27.11:37:42.530>
user = 'https://github.com/mcc-devel'
```

bugs.python.org fields:
```python
activity = <Date 2021-08-27.11:37:42.530>
actor = 'lukasz.langa'
assignee = 'none'
closed = True
closed_date = <Date 2021-08-27.11:37:42.531>
closer = 'lukasz.langa'
components = ['Build']
creation = <Date 2021-08-25.10:03:29.733>
creator = 'jett8998'
dependencies = []
files = []
hgrepos = []
issue_num = 44997
keywords = ['patch']
message_count = 13.0
messages = ['400249', '400257', '400268', '400308', '400318', '400320', '400366', '400367', '400412', '400416', '400417', '400418', '400419']
nosy_count = 6.0
nosy_names = ['ronaldoussoren', 'ned.deily', 'lukasz.langa', 'miss-islington', 'erlendaasland', 'jett8998']
pr_nums = ['27979', '27994', '27995', '27996']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = 'compile error'
url = 'https://bugs.python.org/issue44997'
versions = ['Python 3.8', 'Python 3.9', 'Python 3.10', 'Python 3.11']
```

</p></details>

**Comment by eugeneyarovoi** [comment]
If I'm experiencing this issue on a Mac, what do I need to do to get this patch? Do I need to install something, use a different version of pysqlite3?

**Comment by erlend-aasland** [comment]
> If I'm experiencing this issue on a Mac, [...]

Please open a new issue and describe what problems you are experiencing.

> [...] what do I need to do to get this patch?

Nothing apart from using recent CPython sources; the PR was merged to `main` two years ago.

> Do I need to install something, use a different version of pysqlite3?

`pysqlite3` is an external project. This issue is about the sqlite3 stdlib extension module. There is no need to install anything.

> I see a fix was submitted, but where?

To `main` (3.11 development at that time) and backported to 3.10, 3.9 and 3.8. This means the fix is present in Python 3.8 and newer.
